io.openems.edge.battery.soltaro/src/io/openems/edge/battery/soltaro/single/versionc/statemachine/ErrorHandling.java

```diff
@@ -65,8 +65,8 @@ protected void onEntry(Context context) throws OpenemsNamedException {
 
 	@Override
 	protected void onExit(Context context) throws OpenemsNamedException {
-		context.component.setMaxStartAttempts(false);
-		context.component.setMaxStopAttempts(false);
+		context.component._setMaxStartAttempts(false);
```

Comment:
Is `setMaxStartAttemptsState` ok? Could be a 'soft' guideline to postfix all StateChannels (no matter if info, warning or fault) with 'State' to make that clear. E.g. MAX_START_ATTEMPTS_STATE. This would allow to change the severity later without renaming the Channel.